tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -108,6 +97,53 @@ def run(**kw):
             f"{kw.get('ceph_cluster_dict').values} has less or more clusters Than Expected(2 clusters expected)"
         )
         log.exception(ve)
+        return 1
     except Exception as e:
         log.exception(e)
         return 1
+
+
+def run(**kw):
+    """
+    Snapshot based rbd mirroring schedulers at cluster and pool level.
+
+    Args:
+        **kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Test case covered -
+    CEPH-83575375 and CEPH-83575376
+
+    Test Case Flow:
+    1. Configure snapshot based mirroring between two clusters
+    2. Add mirror snapshot schedule at cluster and pool level
+    3. create some images and wait for them to mirror to secondary
+    4. Check that mirror snapshots are created for each images
+    5. Create a snapshot mirror schedule and list the snapshots
+    6. View the status of schedule snapshot
+    7. remove the snapshot schedule from cluster and pool level, verify the same
+    8. Perform test steps for both Replicated and EC pool
+    """
+    log.info(
+        "Starting CEPH-83575375 and CEPH-83575376, \
+        Create snapshot based RBD mirror at cluster and pool level and verify the same"
+    )
+
+    mirror_obj = rbd_mirror_config(**kw)
+
+    if mirror_obj:
+        log.info("Executing test on replicated pool")
+        if test_snapshot_schedule(
+            mirror_obj.get("rep_rbdmirror"), "rep_pool_config", **kw
+        ):
+            return 1
+
+        log.info("Executing test on ec pool")
```

Comment:
Log did not show any execution on ec pool.

Reply:
sorry it ran against wrong branch
will update success logs soon it's been running
do we have any suggestion apart from log?

Reply:
Attached proper success logs.